src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java

```diff
@@ -79,6 +97,8 @@ public void readAndProcess() {
                 Map<String, Object> data = new HashMap<>();
                 data.put("message", msg.getValue() == null ? "" : new String(msg.getValue()));
                 logger.error(String.format("%s - Failed to read message from queue", id), e, data);
+            } finally {
```

Comment:
i think this is in the wrong place, processing will be set to false after the first message and not set to true again.

Reply:
Moved this.